build.sbt

```diff
@@ -512,8 +513,13 @@ lazy val publishSettings = Seq(
   )
 ) ++ credentialSettings ++ sharedPublishSettings ++ sharedReleaseProcess
 
+
 // These aliases serialise the build for the benefit of Travis-CI.
-addCommandAlias("buildJVM", "catsJVM/test")
+addCommandAlias("buildAlleycatsJVM", ";alleycatsCoreJVM/test;alleycatsLawsJVM/test;alleycatsTestsJVM/test")
+
+addCommandAlias("buildTestingJVM", ";kernelLawsJVM/test;lawsJVM/test;testkitJVM/test;testsJVM/test")
+
+addCommandAlias("buildJVM", ";macrosJVM/test;kernelJVM/test;buildTestingJVM;jvm/test;coreJVM/test;freeJVM/test;buildAlleycatsJVM")
```

Comment:
sorry to nitpick, but this gets harder for me to track what's included in what. maybe we break out a "buildKernelJVM" and a "buildFreeJVM" too for the sake of a clearer structure? Just a thought.

Reply:
Heh.  I originally had: 

> -addCommandAlias("buildJVM", ";macrosJVM/test;kernelJVM/test;kernelLawsJVM/test;lawsJVM/test;testkitJVM/test;testsJVM/test;jvm/test;coreJVM/test;freeJVM/test;alleycatsCoreJVM/test;alleycatsLawsJVM/test;alleycatsTestsJVM/test")

Ha! Not a problem to break up more, it's more confusing than it should be _whatever_ we do, so I thought I should PR _something_ to get some thoughts. I'll update it now.